redi-connect-front/src/components/Notifier.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState, useEffect, useCallback } from 'react'
+import Snackbar from '@material-ui/core/Snackbar'
+
+let openSnackbarFn: ({message, action}: {message: string, action?: any}) => void
```

Comment:
can we maybe in general here a custom hook, simply a suggestion. Something maybe like :
```
export const useNotifier = () => {
  const [isOpen, setOpen] = useState(false)
  const [message, setMessage] = useState('')
  const [action, setAction] = useState(null)

  function open({ message, action }: MessageI) {
    setOpen(true)
    setMessage(message)
    setAction(action)
  }

  function close() {
   setOpen(false)
  }

  function cleanMessage() {
   setMessage('')
  }

  return {
    open,
    close,
    cleanMessage,
    isOpen,
    message,
    action,
  }
};
``` 

And export and use this around?

Reply:
its being used outside of a react component, so hook won't work.